Disable sync_chart (#2742)
kubeapps/kubeapps: docs/developer/ci.md

  - `local_e2e_tests` (always): it runs locally (i.e., inside the CircleCI environment) the e2e tests. Please refer to the [e2e tests documentation](./end-to-end-tests.md) for further information. In this job, before executing the script [`script/e2e-test.sh](../../script/e2e-test.sh), the proper environment is created. Namely:
    - Install the required binaries (kind, kubectl, mkcert, helm).
    - Spin up two Kind clusters.
    - Load the CI images into the cluster.
    - Run the integration tests.
  - `GKE_x_xx_MASTER` and `GKE_X_XX_LATEST_RELEASE` (on master): there is a job for each [Kubernetes version supported by Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/release-notes) (GKE). It will run the e2e tests in a GKE cluster (version X.XX) using either the code in `master` or in the latest released version. If a change affecting the UI is pushed to master, the e2e test might fail here. Use a try/catch block to temporarily work around this.
  - `push_images` (on master): the CI images (which have already been built) get re-tagged and pushed to the `kubeapps` account.

>   - `release` (on tag): it creates a GitHub release based on the current tag by executing the script [script/create_release.sh](../../script/create_release.sh).

Reviewer: I'm assuming the only change in this block is the removal of the `sync_chart` line and switching the asterisks to hyphens.

Author: Yes, it is the only actual change I performed there.
Sorry, just I thought that github was able to detect the change within a line. But yep, I run `prettier` (the same we use for the dashboard code) just to avoid any formatting issues.

I think it's better to send a PR running prettier across all the md files, so that we make clearer the changes.

Note that this process is independent of the release of the official Bitnami images and chart. These Bitnami images will be created according to their internal process (so the Golang, Node or Rust versions we define here are not used by them. Manual coordination is expected here if a major version bump happens to occur).

> Additionally, currently is the Kubeapps team who is in charge of sending a PR to the [chart repository](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) each time a new chart version is pushed to the main branch.

Reviewer: ```suggestion
Additionally, currently the Kubeapps team is responsible for sending a PR to the [chart repository](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) each time a new chart version is pushed to the main branch.
```
